src/main/java/seedu/address/ui/GroupListPanel.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.group.Group;
+
+
+/**
+ * Panel containing the list of groups.
+ */
+public class GroupListPanel extends UiPart<Region> {
+    private static final String FXML = "GroupListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(GroupListPanel.class);
+
+    @FXML
+    private ListView<Group> groupListView;
+
+    /**
+     * Creates a {@code GroupListPanel} with the given {@code ObservableList}.
+     * @param groupList
```

Comment:
Better remove `@param groupList` if there is no description (self-explanatory). Otherwise, put a description there.